### ISSUE TITLE
Annotation wrong when pragma is defined on the same comment line

### ISSUE THREAD

**Issue by karolisgrinkevicius** [rpl/flow-coverage-report#135]
Let's say I have pragma defined at the top of the file:
```js
// @flow @jsx h
```
Running a script below adds annotation `no-flow` which is not correct.
```sh
flow-coverage-report -f ./node_modules/.bin/flow -i src/**/*.js
```

Are you able to fix it?

**Comment by rpl** [comment]
@karolisgrinkevicius the parsing of the @flow pragma is handled by the [flow-annotation-check](https://github.com/ryan953/flow-annotation-check) and so I'm pretty sure that we have to fix this issue in the upstream package and update the dependencies once the fix has been released.

@ryan953 what do you think about this issue?

**Comment by karolisgrinkevicius** [comment]
@rpl you are absolutely correct.

@ryan953 I think regex in https://github.com/ryan953/flow-annotation-check/blob/master/src/promisified.js#L17 has to be modified by ignoring any annotation following `@` except `flow`. Maybe there's a better way to solve it.

**Comment by ryan953** [comment]
@karolisgrinkevicius the bit that detects the annotation is over here: [src/flow.js](https://github.com/ryan953/flow-annotation-check/blob/master/src/flow.js#L47-L67) and here's where flow parses the headers: [src/parsing/parsing_service_js.ml](https://github.com/facebook/flow/blob/15e0cbfe7139eb56a8f796db7b18515aad413d39/src/parsing/parsing_service_js.ml#L174-L238)

Since I can't read that ml code well, a quick check confirms that flow-annotation-check is too strict and flow itself will find multiple annotations on one line:
<img width="581" alt="screenshot 2018-01-18 19 37 34" src="https://user-images.githubusercontent.com/187460/35133829-5722c9c2-fc87-11e7-8096-faf27dbe86d6.png">

So we need a PR, I can put something up tonight

**Comment by karolisgrinkevicius** [comment]
@ryan953 you are awesome. Thanks m8.

**Comment by rpl** [comment]
Thanks @karolisgrinkevicius and @ryan953 for your help to quickly fix this issue!
fixed in 798607d1 (and new integration test added in 30b6ae42)
released on npm as bugfix minor release 0.4.1 🎉
